### DISCUSSION TITLE
Regarding X-Touch Editor software

### DISCUSSION THREAD

**Discussion by yomozingo** [rsjaffe/MIDI2LR#672] [Problems]
I use LR with a Macbook.  I was told to download X-Touch Editor software in order to determine if my X-Touch Compact is set up to
send NRPN codes.  Since the editor is a Windows program are there any other way to approach this?  

Thank you!

**Comment by yomozingo** [comment]
My friend came over with PC laptop and we downloaded and opened editor.  I didn't see any indication of NRPN codes in the program.  Any further help would be greatly appreciated.

**Comment by yomozingo** [comment]
I'm back at trying to figure out this problem.  I dowloaded a profile from a group online.  But the same problem has persisted.  For instance, every time I move the 1st fader MIDILR automatically brings up CC:101 in addition to CC:1.  I am attaching a video of the action.  If I clear all rows and move the 1st fader it brings up both CC:1 and CC:101 (as shown in attached screen shot.)  If anyone can help me fix this I would greatly appreciate it.  I used to use X-Touch Compact with PFixer.  My first X-Touch broke then I discovered MIDILR and bought another X-Touch Compact to use with it.  I really miss the convenience of the sliders.  Thank you

https://user-images.githubusercontent.com/108846843/181291074-72f3ee78-5400-4f03-9144-ed87faa650eb.mov

<img width="563" alt="Screen Shot 2022-07-27 at 11 36 06 AM" src="https://user-images.githubusercontent.com/108846843/181291117-40949e42-3a91-4d19-a5cc-902ba2fdec59.png">
.

**Reply by yomozingo** [reply]
Update.  I tried layer B and that works for assigning the sliders.   Layer B gives CC:128 along with CC:1 but it works.  Layer A continues to give CC:101 along with CC:1.  

I would be happy to just use Layer B but the sliders do not go to custom setting when going from photo to photo.  Is there something specific that I need to do in order to get the sliders to automatically move to each photos particular settings?

Thank you!

**Comment by antzpantz** [comment]
Sorry, I didn't realise that it was a Windows-only program. You'll probably
have to find a PC to initially check the configuration of the controller.
